packages/cra-template-mephisto-review/template/src/ItemRenderer.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { useMephistoReview } from "mephisto-review-hook";
+import { useParams, useHistory } from "react-router-dom";
+import { Button, H2, Card, Elevation } from "@blueprintjs/core";
+import AppToaster from "./components/AppToaster";
+import "./ItemRenderer.css";
+
+function App() {
+  const { id } = useParams();
+  const {
+    data,
+    isFinished,
+    isLoading,
+    submit,
+    error,
+    mode,
+  } = useMephistoReview({ taskId: id });
+  const history = useHistory();
+
+  const confirmReview = () => {
+    if (mode === "OBO") {
+      history.push("/");
+    } else {
+      AppToaster.show({ message: "Review Sent!" });
+    }
+  };
+
+  const reviewError = (error) => {
+    AppToaster.show({ message: `ERROR: ${error}` });
+  };
+
+  return (
+    <main className="Item-Renderer">
+      {mode === "ALL" ? (
+        <Button
+          onClick={() => history.push("/")}
+          intent="primary"
+          large={true}
+          icon="caret-left"
+          className="home-button"
+        >
+          <b>BACK</b>
+        </Button>
+      ) : null}
+      {error && <h5 className="error">Error: {JSON.stringify(error)}</h5>}
+      {isLoading ? (
+        <h1>Loading...</h1>
+      ) : isFinished ? (
+        <h1>Done reviewing! You can close this app now</h1>
+      ) : (
+        <>
+          <H2>Please review the following data:</H2>
+          <Card className="item" elevation={Elevation.TWO}>
+            <pre>{(data && data.data) || data}</pre>
```

Comment:
you don't need `|| data` anymore

Reply:
Tested changing it to data && data.data and it breaks when the only valid object is data. However I am considering changing the endpoint on review_server.py to always return a data.data and data.id in the data portion of the response. Thoughts?

Reply:
ah you're right, thanks for testing this. && returns the first falsey value, so if data.data is null, it would evaluate to null

if we were to use || instead, then the first truthy value would be returned, which is also not quite what we want because data would always be true.

If data is guaranteed to always be defined, then we could do `data.data || data`, but idk if that assumption is true.

changing the endpoint could also be good. it would def simplify the usage a bit here

Reply:
I opted to change the endpoints for now such that data.data is guaranteed for most responses to improve consistency throughout the app. For now that means there's no use for the OR clause however maybe we'll change that in the future.